src/I18n/En.elm

```diff
@@ -18,9 +19,15 @@ enStrings key =
         GuideTitle ->
             "[cCc] Guide"
 
+        PageTitle string ->
```

Comment:
We should use the page title value here rather than the slug.

Reply:
Of course!